### PR TITLE
libxcb should require libpthread-stubs

### PR DESCRIPTION
**Pull request by Mno-hime**
https://github.com/OpenIndiana/oi-userland/pull/3147#issuecomment-304537975
Supersedes https://github.com/OpenIndiana/oi-userland/pull/3239.

### CODE REVIEW

File: components/x11/libxcb/Makefile [hunk 1]
```diff
@@ -78,6 +79,7 @@ test: $(NO_TESTS)
 # Build dependencies
 REQUIRED_PACKAGES += developer/check
 REQUIRED_PACKAGES += x11/header/xcb-protocols
+REQUIRED_PACKAGES += x11/libpthread-stubs
```
Comment (alarcher) [inline components/x11/libxcb/Makefile]: x11/library/libpthread-stubs :)

File: components/x11/libxcb/libxcb.p5m [hunk 2]
```diff
@@ -20,6 +20,8 @@ set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
 set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
+depend type=require fmri=x11/libpthread-stubs
```
Comment (alarcher) [inline components/x11/libxcb/libxcb.p5m]: also x11/library/libpthread-stubs :)

Comment (Mno-hime) [reply components/x11/libxcb/libxcb.p5m]: I am sorry for my incompetence :).

**Review by alarcher — APPROVED** [review]

File: components/x11/libxcb/libxcb.p5m (hunk 2 above)
Comment (alarcher) [reply components/x11/libxcb/libxcb.p5m]: Hi hi :)

**Pull request merged**